tests/main/preseed-lxd/task.yaml

```diff
@@ -0,0 +1,102 @@
+summary: Check that preseeding of current ubuntu cloud image works under lxd.
+description: |
+  This test checks that preseeding of Ubuntu cloud images with snap-preseed
+  command works in lxc container. 
+
+# this test works only on 18.04 because it requires lxd from deb (lxd snap
+# as it wouldn't allow mount) and tries to replicate launchpad builder setup.
+systems: [ubuntu-18.04-*]
+
+environment:
+  IMAGE_MOUNTPOINT: /mnt/cloudimg
+
+prepare: |
+  IMGURL="https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img"
```

Comment:
you could use the one provided by nested: https://github.com/snapcore/snapd/blob/master/tests/lib/nested.sh#L85
Those images are updated weekly.

Reply:
Right, done (note, I'm not using the helper because I need different ubuntu than implied by SPREAD_SYSTEM).